### DISCUSSION TITLE
Sponge Port

### DISCUSSION THREAD

**Discussion by ryandis44** [Zrips/CMI#4446] [Feature Requests]
I have been using CMI for a while now and I absolutely love it. It is a very well written plugin and the migration from essentials was seamless...this is sounding more like a review...

I would like to see a Sponge version of CMI, as I have started up a modded server and the only "essentials" plugin out there is Nucleus. Nucleus is not a bad plugin, but I personally like CMI better.

**Comment by CloudeLecaw** [comment]
a sponge port wil not hapen so fast. personally i would say this will never be happen^^
